module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -482,9 +482,25 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             $itemResponse = $this->makeRequest('GET', '/item-storage/items', $query);
             $itemBody = json_decode($itemResponse->getBody());
             $notesFormatter = function ($note) {
-                return $note->note ?? '';
+                return !$note->staffOnly && $note->note ? $note->note : '';
             };
+            $textFormatter = function ($supplement) {
+                $format = '%s %s';
+                $supStat = $supplement->statement;
+                $supNote = $supplement->note;
+                $statement = sprintf($format, $supStat, $supNote);
```

Comment:
Is there always going to be a statement and a note? If not, it might be worth wrapping a `trim` around this to kill any superfluous whitespace.

Reply:
Great suggestion!